website/src/pages/StatusPage/components/PreviousExecutions/Columns.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ColumnDef } from "@tanstack/react-table";
+import { Badge, Variant } from "@/components/ui/badge";
+import { formatDistanceToNow, format } from "date-fns";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+
+export type PreviousExecution = {
+  uuid: string;
+  source: string;
+  git_ref: string;
+  status: string;
+  type_of: string;
+  pull_nb: string;
+  golang_version: string;
+  started_at: string;
+  finished_at: string;
+};
+
+export const columns: ColumnDef<PreviousExecution>[] = [
+  {
+    header: "UUID",
+    accessorKey: "uuid",
+    cell: ({ row }) => {
+      const formatted = row.original.uuid.slice(0, 8);
+      return <div>{formatted}</div>;
+    },
+  },
+  {
+    header: "SHA",
+    accessorKey: "git_ref",
+    cell: ({ row }) => {
+      const formatted = row.original.git_ref.slice(0, 8);
+      return (
+        <div>
+          {" "}
+          <Badge color="primary">{formatted}</Badge>
+        </div>
+      );
+    },
+  },
+  {
+    header: "Source",
+    accessorKey: "source",
+    filterFn: (row, id, value) => {
+      return value.includes(row.getValue(id));
+    },
+  },
+  {
+    header: "Workload",
+    accessorKey: "type_of",
```

Comment:
Should be `workload`

```suggestion
    accessorKey: "workload",
```